pkg/certs/ensure.go

```diff
@@ -131,6 +137,10 @@ func EnsureCerts(
 			Name:            secretName,
 			Namespace:       currentNamespace,
 			OwnerReferences: ownerRef,
+			Labels: map[string]string{
+				CertSecretLabelAppKey:     CertSecretLabelAppValue,
+				CertSecretLabelReleaseKey: currentNamespace,
```

Comment:
I don't think the release name is equal to the namespace name, that only happens for development where the release is equal the namespace

Reply:
I have made the necessary changes.